commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java

```diff
@@ -266,6 +270,14 @@ public List<String> getMetricsHostAllowList() {
     return metricsHostAllowList;
   }
 
+  @Override
+  public Optional<MetricsPushOptions> getMetricsPushOptions() {
+    if (metricsPushOptions.isMetricsPushEnabled()) {
```

Comment:
From memory, the mixin might have some weird side effects if they are not specified on command line. Make sure that `metricsPushOptions` is not `null` before calling `isMetricsPushEnabled`

Reply:
However, if there is an AT which is testing the default behavior of metrics push disabled, then nothing to change here.